packages/presentation/lib/app/widgets/bottom_navigation.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:presentation/app/app_bloc.dart';
+import 'package:presentation/utils/colors.dart';
+import 'package:presentation/utils/dimens.dart';
+import 'package:presentation/utils/images/paths.dart';
+
+class BottomNavigation extends StatelessWidget {
+  const BottomNavigation({
+    Key? key,
+    required this.bloc,
+  }) : super(key: key);
+
+  final AppBloc bloc;
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = AppLocalizations.of(context)!;
+    return Container(
+      padding: const EdgeInsets.only(bottom: Dimens.size14),
+      decoration: const BoxDecoration(
+        border: Border(
+          top: BorderSide(
+            width: Dimens.size05,
+            color: PrimaryColors.primaryUnselected,
+          ),
+        ),
+      ),
+      child: BottomNavigationBar(
+        showSelectedLabels: false,
+        showUnselectedLabels: false,
+        backgroundColor: PrimaryColors.primaryBackgroundColor,
+        elevation: 0,
+        items: <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+            icon: SvgPicture.asset(
+              ImagesPath.movieIcon,
+              color: PrimaryColors.whiteWithOpacity80,
+            ),
+            activeIcon: SvgPicture.asset(
+              ImagesPath.movieIcon,
+              color: PrimaryColors.primarySelected,
+            ),
+            label: l10n.title,
+          ),
```

Comment:
if u will create the wrapper for BottomNavigationBarItem widget, u will have an option to add widgets in more suitable way, like

Item(
  iconPath: "someImage.svg".
  label: "title"
)

and in the wrapper widget u will form the widget as u do it now, since colors for icon and activeIcon are the same for all states

or u can use such variant - https://stackoverflow.com/a/57126622/10597533. it looks even prettier